lib/l10n/intl_it.arb

```
@@ -411,5 +411,10 @@
   "spanish": "Spagnolo",
   "italian": "Italiano",
   "chooseLanguageDescription": "Scegli la tua lingua preferita o usa il predefinito di sistema",
-  "done": "FATTO"
+  "loadingOrder": "Caricamento ordine...",
+  "done": "FATTO",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)